src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1912,7 +1912,29 @@ bool isRefEscape
         {
             if (receiver is { })
             {
-                var tuple = getReceiver(symbol, receiver);
+                var method = symbol switch
+                {
+                    MethodSymbol m => m,
+                    PropertySymbol p => p.GetMethod ?? p.SetMethod,
+                    _ => throw ExceptionUtilities.UnexpectedValue(symbol)
+                };
+
+                if (receiver is not BoundValuePlaceholderBase && method is not null && receiver.Type!.IsVerifierValue())
```

Comment:
nit: consider changing IsVerifierValue to IsValueType here.

It looks like an implicit defensive copy can still occur when `receiver.Type` is a type parameter. [SharpLab](https://sharplab.io/#v2:EYLgtghglgdgPgAQMwAJYBcCmAnAZhAY0xQEkBYAKAG9KU6UEAWFAWQAoBKAbkoF9LKyBgCYUAYQA8AFQB8KAO4ALHMSkoQKAM7psAVwLoANKRQ0K9FNkwQAJgHsYAGwCeKNeh7n6Qpq06naC3p0ADp2bkC6fgpeIA==). But since we check `IsVerifierValue()` instead of `IsValueType`, we won't create a placeholder for it in this analysis pass.

It looks like a ref to such an implicit local can't escape in this case, because you can't call an `[UnscopedRef]` method on a receiver of type parameter type. However, it seems like it would be more robust to create the placeholder anyway, even if it will never escape under the current language rules. Eventually we might make it possible to add `[UnscopedRef]` to an interface method, for example, in which case the right behavior will just fall out.